ci/requirements-py36.yml

```diff
@@ -18,3 +18,4 @@ dependencies:
   - pip:
     - coveralls
     - pytest-cov
+    - pydap
```

Comment:
Our pydap tests are currently network only tests, and we only run those on Python 3.6.